.github/workflows/build-kind-image.yaml

```
@@ -58,7 +61,10 @@ jobs:
         uses: docker/setup-buildx-action@v3
       - name: Build KinD image for ${{ matrix.arch }}
         run: |
-          kind build node-image ${{ github.event.inputs.tag < 'v1.31' && '--type source .' || format('--type release {0}', github.event.inputs.tag) }} --arch ${{ matrix.arch }} --image ghcr.io/${{ github.repository_owner }}/kind-node:${{ github.event.inputs.tag }}-${{ matrix.arch }}
+          kind build node-image ${{ github.event.inputs.tag < 'v1.31' && '--type source .' || format('--type release {0}', github.event.inputs.tag) }} \
+          --arch ${{ matrix.arch }} \
+          --image ghcr.io/${{ github.repository_owner }}/kind-node:${{ github.event.inputs.tag }}-${{ matrix.arch }} \
+          ${{ inputs.base_image && format('--base-image {0}', inputs.base_image) || '' }}
```

Not sure if this needs to a string comparison?

```suggestion
          ${{ inputs.base_image != '' && format('--base-image {0}', inputs.base_image) || '' }}
```